model/auth.ts

```diff
@@ -0,0 +1,10 @@
+export class Credentials {
```

Comment:
If these aren't being instantiated they can be types instead of classes
`export type`

Also if you are importing just a type you should use
`import type xxx`